int32 drone_id_from

FrontierClusterMsg[] frontier_list

